src/scss/components/_add_post.scss

```diff
@@ -0,0 +1,5 @@
+.write-post{
+    width: 75vw;
```

Comment:
I'd prefer for you do do this using row/col classes

Reply:
Fixed!